web/book/src/reference/syntax/literals.md

```diff
@@ -27,92 +27,24 @@ select {
 }
 ```
 
-## Strings
```

Comment:
Can we put a link to the Strings page here, since the normal string is a literal?